corehq/motech/repeaters/tests/test_models.py

```diff
@@ -229,38 +229,45 @@ def set_next_attempt_at(repeater, when):
         repeater.save()
 
 
-class ResponseMock:
+class Unknown:
     pass
 
 
 class IsResponseTests(SimpleTestCase):
 
     def test_has_text(self):
-        resp = ResponseMock()
-        resp.text = '<h1>Hello World</h1>'
+        resp = Unknown()
+        resp.text = '<h1>Hello World</h1>'  # type: ignore
```

Comment:
What does `# type: ignore` do? I assume it's a type hint, but do we have any automation that cares about that?

Reply:
`# type: ignore` tells IDEs and mypy not to give the developer type warnings -- in this case specifically not to warn them that they are assigning a value to an undefined attribute. It is the equivalent of `# flake8: noqa`.

Reply:
Could it be avoided by adding `text = ""` to the `Unknown` class?

Reply:
Comments dropped in 68e39da35de

> Could it be avoided by adding `text = ""` to the `Unknown` class?

That would resolve the warnings, but it would cause some of the tests for `is_response()` to fail, because it would make the instance of `Unknown` look like a response.